FusionIIIT/Fusion/settings/development.py

```diff
@@ -9,10 +9,10 @@
 DATABASES = {
      'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'fusionlab',
-        'HOST': 'localhost',
-        'USER': 'fusion_admin',
-        'PASSWORD': 'hello123',
+        'NAME': 'fusiondb',
+        'HOST': os.environ.get("DB_HOST"),
+        'USER': 'fusionuser',
+        'PASSWORD': 'password',
```

Comment:
```suggestion
        'NAME': 'fusiondb',
        'HOST': os.environ.get("DB_HOST"),
        'USER': 'fusionuser',
        'PASSWORD': 'password',
```

@yatharthagoenka Kindly remove these changes.